src/test/java/uk/gov/companieshouse/acsp/manage/users/integration/UserAcspMembershipIntegrationTest.java

```diff
@@ -83,7 +83,22 @@ void setup() {
     removedMembership.setRemovedBy("admin3");
     removedMembership.setEtag("etag2");
 
-    acspMembersRepository.saveAll(List.of(activeMembership, removedMembership));
+    AcspMembersDao acspMembersDao = new AcspMembersDao();
+    acspMembersDao.setId("acsp1");
+    acspMembersDao.setUserId("user123");
+    acspMembersDao.setAcspNumber("ACSP123");
+    acspMembersDao.setUserRole(AcspMembership.UserRoleEnum.ADMIN);
+    acspMembersDao.setCreatedAt(LocalDateTime.now().minusDays(10));
+    acspMembersDao.setAddedAt(LocalDateTime.now().minusDays(10));
```

Comment:
You declared on line 42, so you could use it here:

```java
private final LocalDateTime now = LocalDateTime.now();
```

```suggestion
    acspMembersDao.setCreatedAt(now.minusDays(10));
    acspMembersDao.setAddedAt(now.minusDays(10));
```